lib/config.py

```diff
@@ -4,6 +4,7 @@
 import os
 import sys
 
+from lib.parameter_store import ParameterStore
```

Comment:
Aha, so ParameterStore gets the "environment" of the lambda function. Looks good

Reply:
Correct, see `template.yml` (used for deployment) this sets the environment variable [1] and creates the config in SSM. [2]

1. https://github.com/wesselt/bunq2ynab/pull/22/commits/38607aefd4b9246ce23fe271f2aa062c3dc3ac9a#diff-363d481fe07d1094db2338998d381b71R46-R47
2. https://github.com/wesselt/bunq2ynab/pull/22/commits/38607aefd4b9246ce23fe271f2aa062c3dc3ac9a#diff-363d481fe07d1094db2338998d381b71R64-R80